x-pack/plugins/ml/public/application/jobs/new_job/common/job_creator/util/model_memory_estimator.ts

```diff
@@ -42,23 +45,36 @@ export const modelMemoryEstimatorProvider = (jobValidator: JobValidator) => {
         // clone the object to compare payloads and proceed further only
         // if the configuration has been changed
         map(cloneDeep),
+        tap(v => {
+          // eslint-disable-next-line no-console
+          console.log('Clone object of the incoming config: ', JSON.stringify(v, null, 2));
+        }),
         distinctUntilChanged(isEqual),
-        switchMap(payload => {
-          const isPayloadValid = jobValidator.isModelMemoryEstimationPayloadValid;
-
-          return isPayloadValid
-            ? ml.calculateModelMemoryLimit$(payload).pipe(
-                pluck('modelMemoryLimit'),
-                catchError(error => {
-                  // eslint-disable-next-line no-console
-                  console.error('Model memory limit could not be calculated', error.body);
-                  error$.next(error.body);
-                  return of(DEFAULT_MODEL_MEMORY_LIMIT);
-                })
-              )
-            : of(DEFAULT_MODEL_MEMORY_LIMIT);
+        tap(v => {
+          // eslint-disable-next-line no-console
+          console.log('New config: ', JSON.stringify(v, null, 2));
```

Comment:
these log messages are adding a lot of information to the console. are they still needed?

Reply:
I still need them for checking the flaky test runner, will remove after